ui/StatusQ/src/StatusQ/Controls/StatusBaseInput.qml

```diff
@@ -436,7 +436,7 @@ Item {
         id: clearButton
 
         StatusFlatRoundButton {
-            visible: edit.text.length != 0 && root.clearable && !root.multiline
+            visible: edit.length != 0 && root.clearable && !root.multiline
```

Comment:
It's not the same, from the docu:

> As this number doesn't include any formatting markup it may not be the same as the length of the string returned by the [text](https://github.com/status-im/status-desktop/pull/9574/qml-qtquick-textedit.html#text-prop) property.